static/app/components/events/meta/annotatedText/index.tsx

```diff
@@ -5,13 +5,24 @@ type Props = {
   value: React.ReactNode;
   className?: string;
   meta?: Record<any, any>;
+  withOnlyFormattedText?: boolean;
 };
 
-export function AnnotatedText({value, meta, className, ...props}: Props) {
+export function AnnotatedText({
+  value,
+  meta,
+  className,
+  withOnlyFormattedText = false,
```

Comment:
Are you adding this prop to remove the formatting so that you can apply error states to the whole key/value row? I'm not sure if that's the best idea since the annotated text is sometimes only applied to sections of text and not the entire value.  It also seems weird to keep a tooltip but not have the highlighted text so you know where to hover.

Such as for size limits, how would this be handled? 

![CleanShot 2024-04-15 at 09 43 07](https://github.com/getsentry/sentry/assets/10888943/1fae5972-513f-4132-8aad-86e10df70547)

Reply:
I didn't realize this was something we did. At least as per designs, the background was removed for the `<invalid>`/`<redacted>` values which was my intention, but you're right that if partially redacted text has a tooltip, we would want to retain it. I'll test this locally and update.

Reply:
IMO you might want to sync with Vu and see if this requirement can be modified. It would be a lot simpler and more consistent to just keep the annotations the same as they are in the rest of the page, so if that's an option that would be my preference.